executor/src/lib.rs

```diff
@@ -136,6 +136,9 @@ pub struct TransactionData {
 
     /// The number of newly created accounts.
     num_account_created: usize,
+
+    /// The transaction info hash if the VM status output was keep, None otherwise
+    txn_info_hash: Option<HashValue>,
```

Comment:
This is something that can be derived from the transaction and other fields in this struct, so we don't necessarily need to store this here?

Reply:
https://github.com/libra/libra/blob/714347d80df1917f37a22c815e7864da21e632bf/executor/src/block_processor.rs#L558
https://github.com/libra/libra/blob/714347d80df1917f37a22c815e7864da21e632bf/executor/src/block_processor.rs#L582

We would need to know status.major_status and  vm_output.status(), we should be able to regenerate the transaction from the block though.